src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/coverage/CoverageRenderer.kt

```diff
@@ -26,8 +30,20 @@ import java.awt.event.MouseEvent
  * @param color color of marker
  * @param lineNumber lineNumber to color
  * @param tests list of tests that cover this line
+ * @param coveredMutation list of mutant operation which are covered by tests
+ * @param coveredNotMutation list of mutant operation which are not covered by tests
+ * @param mapMutantsToTests map of mutant operation -> List of names of tests which cover mutant
+ * @param project the current project
  */
-class CoverageRenderer(private val color: Color, private val lineNumber: Int, private val tests: List<String>, private val project: Project) :
+class CoverageRenderer(
+    private val color: Color,
+    private val lineNumber: Int,
+    private val tests: List<String>,
+    private val coveredMutation: List<String>,
+    private val coveredNotMutation: List<String>,
```

Comment:
Pperhaps a name like `notCoveredMutation` would be a bit less confusing :stuck_out_tongue:

Reply:
Done!